src/drm.cpp

```diff
@@ -469,16 +462,39 @@ int Device::getResources() {
     objects_[obj.id()] = &obj;
   }
 
-  /* Finally, perform all delayed setup of mode objects. */
+  performAllDelayedSetupOfModeObjects(objects_);
```

Comment:
The old code did not return zero in all cases:

`return performAllDelayedSetupOfModeObjects(objects_);`

would work though.

Reply:
Thanks for the feedback Eric, I'll change this